setup.py

```diff
@@ -37,7 +37,7 @@
     setup_requires=["setuptools_scm"],
     install_requires=[
         "mlflow @ git+https://github.com/mlflow/mlflow.git#egg=mlflow",
-        "faculty==0.23.0.dev1",
+        "faculty @ git+https://github.com/facultyai/faculty.git@restore-experiment#egg=faculty",
```

Comment:
- [ ] change back to a tagged version when https://github.com/facultyai/faculty/pull/91 merged